CRM/Event/Form/ManageEvent/EventInfo.php

```diff
@@ -177,8 +172,8 @@ public function buildQuickForm() {
     $this->addElement('checkbox', 'is_share', ts('Allow sharing through social media?'));
     $this->addElement('checkbox', 'is_map', ts('Include Map to Event Location'));
 
-    $this->addDateTime('start_date', ts('Start Date'), FALSE, array('formatType' => 'activityDateTime'));
-    $this->addDateTime('end_date', ts('End Date / Time'), FALSE, array('formatType' => 'activityDateTime'));
+    $this->add('datepicker', 'start_date', ts('Start Date'), [], FALSE, ['time' => TRUE]);
+    $this->add('datepicker', 'end_date', ts('End Date / Time'), [], FALSE, ['time' => TRUE]);
```

Comment:
Scope creep: these form labels are inconsistent. Why mention time in one but not the other?
What about simply "Start" and "End"?